backend/qemu.pm

```diff
@@ -166,28 +166,27 @@ sub save_memory_dump {
                 uri => sprintf("exec:gzip -c > ulogs/%s-vm-memory-dump.gz", $args->{filename}),
             }});
 
-    die(sprintf("Migration failed: desc: %s, class: %s, stopped", $rsp->{error}->{desc}, $rsp->{error}->{class})) if ($rsp->{error});
-
+    bmwqemu::logdie(sprintf("Migration failed: desc: %s, class: %s, stopped", $rsp->{error}->{desc}, $rsp->{error}->{class})) if ($rsp->{error});
 
     do {
 
         sleep 0.5;    #We want to wait a decent amount of time, a file of 1GB will be
                       # migrated in about 40secs with an ssd drive. and no heavy load.
         $rsp = $self->handle_qmp_command({execute => "query-migrate"});
 
-        diag "Migrating total bytes:     \t" . $rsp->{return}->{ram}->{total};
-        diag "Migrating remaining bytes:   \t" . $rsp->{return}->{ram}->{remaining};
+        bmwqemu::diag "Migrating total bytes:     \t" . $rsp->{return}->{ram}->{total};
```

Comment:
@aaannz Mostly 'cause i wanted to know which lines/files i have touched... See my last comment also.